.github/workflows/Format.yml

```diff
@@ -0,0 +1,31 @@
+name: format-pr
```

Comment:
Most of our CI pipeline names starts with capital letters, maybe change it as `Format-PR` or `JuliaFormatter-PR` ?

Reply:
If we are using `JuliaFormatter-PR` as the name, maybe also change the filename as `JuliaFormatter.yml` ?